src/category_theory/limits/shapes/equalizers.lean

```diff
@@ -156,4 +171,51 @@ def cofork.of_cocone
 @[simp] lemma cofork.of_cocone_ι {F : walking_parallel_pair.{v} ⥤ C} (t : cocone F) (j) :
   (cofork.of_cocone t).ι.app j = eq_to_hom (by tidy) ≫ t.ι.app j := rfl
 
+variables (f g)
+
+section
+variables [has_limit (parallel_pair f g)]
+
+abbreviation equalizer := limit (parallel_pair f g)
+
+abbreviation equalizer.ι : equalizer f g ⟶ X :=
+limit.π (parallel_pair f g) zero
+
+abbreviation equalizer.w : equalizer.ι f g ≫ f = equalizer.ι f g ≫ g :=
```

Comment:
Can/should we `reassoc` this abbreviation?

Reply:
Yes. In fact, it should just be a simp lemma (as a lemma, rather than a definition, there's not point making it an abbreviation because we'll never want to unfold it).

I've also renamed this `equalizer.condition`, per the discussion above.